src/app/shared/services/utils/utils.service.spec.ts

```diff
@@ -131,4 +132,89 @@ describe('Utils service', () => {
     color = '#000000';
     expect(Utils.isColorDark(color)).toBeTruthy();
   });
+
+  it('should check if CIDR valid', () => {
+    const validValues = [
+      // IP v4
+      '99.198.122.146/32',
+      '46.51.197.88/8',
+      '173.194.34.134/12',
+      '0.0.0.0/0',
+      // IP v6
+      'fe80:0000:0000:0000:0204:61ff:fe9d:f156/100',
+      '::1/128',
+      'a:b:c:d:e:f:0::/64',
+      'FE80::/10'
+    ];
+    const invalidValues = [
+      'invalid',
+      '',
+      ' ',
+      '0.0.0.0',
+      '::/',
+      null,
+      // IP v4
+      '.100.100.100.100/16',
+      '100.100.100.100./32',
+      'http://123.123.123/28',
+      '1000.2.3.4/14',
+      // IP v6
+      'fe80:0000:0000:0000:0204:61ff:fe9d:f156/129',
+      '1::5:400.2.3.4/64',
+      '2001:0000:1234:0000:0000:C1C0:ABCD:0876  0/64',
+      '1.2.3.4::/64',
+      ':::/64',
+      '::2222:3333:4444:5555:7777:8888::/64'
+    ];
+
+    for (const value of validValues) {
+      expect(Utils.cidrIsValid(value)).toBe(true);
+    }
+    for (const value of invalidValues) {
+      expect(Utils.cidrIsValid(value)).toBe(false);
+    }
+  });
+
+  it('should return proper CIDR type', () => {
+    const validV4Values = [
```

Comment:
Move addresses const declarations out of test to the root of the file to avoid code duplication